### PR TITLE
Added active servers to statsz.

### PR DESCRIPTION
**Pull request by derekcollison**
This is generally useful but will also help with cli actions since we know from first response how many total responses are expected.

Signed-off-by: Derek Collison <derek@nats.io>

/cc @nats-io/core

### CODE REVIEW

File: server/events.go [hunk 1]
```diff
@@ -924,12 +931,6 @@ func (s *Server) remoteServerUpdate(sub *subscription, _ *client, subject, reply
 	}
 	si := ssm.Server
 	node := string(getHash(si.Name))
-	if _, ok := s.nodeToInfo.Load(node); !ok {
```
Comment (matthiashanel) [inline server/events.go:927]: Is this compensated by the addition in processNewServer?

**Comment by ripienaar** [comment]
LGTM, didnt check how this is maintained - like how accurate it is and on what time frame but will help with hints

File: server/events.go (hunk 1 above)
Comment (derekcollison) [reply server/events.go:927]: Yes.

**Comment by derekcollison** [comment]
If the servers shutdown and that message is received the server is removed, otherwise we will remove it when we sweep for lost remotes, iirc 2mins.

**Review by matthiashanel — APPROVED** [review]
LGTM

**Pull request merged**